.github/workflows/format.yml

```diff
@@ -0,0 +1,49 @@
+name: Format
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          access_token: ${{ github.token }}
```

Comment:
`styfle/cancel-workflow-action`'s `access_token` is already defaulting to `github.token`
https://github.com/styfle/cancel-workflow-action/blob/dee858fff096b2cbcd4f845ce21871c50c5af452/action.yml#L15-L17

```suggestion
```

Reply:
oh cool! i just used the example in the readme 🤷